xtask/src/main.rs

```diff
@@ -0,0 +1,35 @@
+use xshell::pushd;
+use xtask::{
+    build::{self, Target as BuildTarget},
+    runner::{self, Target as RunTarget},
+    EnvArgs, Result, ENV_LIST,
+};
+
+fn main() -> Result<()> {
+    let mut args = std::env::args().skip(1);
+    let command = args.next().unwrap_or_default();
+    let subcommand = args.next().unwrap_or_default();
+
+    let env_parts = std::env::vars()
+        // TODO: Is this really needed?
```

Comment:
Idea to save memory usage, probably not worth it for needing to maintain `ENV_LIST`.